Package.swift

```diff
@@ -22,7 +22,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/IBDecodable/IBDecodable.git", .branch("master")),
-        .package(url: "https://github.com/Carthage/Commandant.git", from: "0.15.0"),
+        .package(url: "https://github.com/Carthage/Commandant.git", .upToNextMinor(from: "0.16.0")),
```

Comment:
Since Commandant 0.17.0, antitypical/Result is replaced to stdlib's.
https://github.com/Carthage/Commandant/pull/150

So I pinned version to 0.16.x at this point.